src/main/java/me/mooy1/infinityexpansion/Items.java

```diff
@@ -1,429 +1,429 @@
 package me.mooy1.infinityexpansion;
 
 import io.github.thebusybiscuit.slimefun4.utils.HeadTexture;
+import io.github.thebusybiscuit.slimefun4.utils.LoreBuilder;
 import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
 import org.bukkit.Material;
 import org.bukkit.enchantments.Enchantment;
-import io.github.thebusybiscuit.slimefun4.utils.LoreBuilder;
 import org.bukkit.inventory.meta.ItemMeta;
 
 public final class Items {
 
     //Machines
 
     public static final SlimefunItemStack QUARRY = new SlimefunItemStack(
-            "QUARRY",
-            Material.STONE_BRICKS,
-            "&7&lQuarry",
-            LoreBuilder.speed(1),
-            LoreBuilder.powerPerSecond(4_000),
-            ""
+        "QUARRY",
+        Material.STONE_BRICKS,
+        "&7&lQuarry",
+        LoreBuilder.speed(1),
```

Comment:
i added another lore here, and at 27 36 45

Reply:
They arent there though
Idk what you mean in the current version they arent there and in my changes are the exact same as the old version just formatted

Reply:
Oh you updated it ah will fix when home